### PR TITLE
Add Decap

### PR DESCRIPTION
**Pull request by grugnog**
This adds https://decapcms.org/ - it is pointing at my fork of this repo for now, so we can play around with PRs etc without generating a bunch of noise.

Some notes:

- We need to use frontmatter "title" field rather than # style headers. As far as I can tell though, apart from the Github pages looking a bit naff everything else seems to work fine.
- The config can handle nested directory trees OK, but there are 2 issues:
    - The configuration requires a sub-directory for content, so I don't think we can just point it at the root directory (at least, I couldn't get it to work!) - so I was able to work around this by just listing each top-level directory separately, using some YAML aliases to avoid duplicating too much. So, we will need to remember to update this if we change any paths but other than that I think it is workable.
    - The file tree uses an README.md in each nested directory to identify the folder name - there might be another way of setting this up, but I think that would actually be nice for us to adopt anyway. This part isn't done, so expect the folder names to be wonky.
- I was able to inject some remark formatter configuration that makes it closer to our canonical format, but there will inevitably be some whitespace diffs in PRs generated by this - for example prettier wants a line after frontmatter and Decap doesn't (and I can't find a way to configure this). Not a major blocker, since these will be fixed by the auto pre-commit fixer.

----
:books: Documentation preview :books:: https://civicactions-handbook--1123.org.readthedocs.build/en/1123/

### CODE REVIEW

**Comment by grugnog** [comment]
@dmundra @ContentBrewmaster starting point if you want to play: https://civicactions-handbook--1123.org.readthedocs.build/en/1123/admin/

**Comment by grugnog** [comment]
The reviewdog fail is [fixed upstream](https://github.com/reviewdog/reviewdog/commit/747bf62aa1f260d46d7f602cadf3c18e7e1dc40a) but still not released, so thinking we just ignore that check here for now.

**Comment by dmundra** [comment]
Thanks @grugnog. I tried logging into https://civicactions-handbook--1123.org.readthedocs.build/en/1123/admin/#/ and I got:

```
Make sure you've configured the API credentials for Github from the Access part of your Netlify site dashboard.
```

I did log into netlify to create my account there but I still get the error. Do we need to be added your fork?

**Comment by dmundra** [comment]
Thanks @grugnog the code changes make sense. Looking forward to giving it a whirl.

**Comment by grugnog** [comment]
@dmundra Decap login still isn't working, but based on the demo yesterday & if the preview site looks good, I think we are OK to merge this?

File: practice-areas/accessibility/accessibility-how-to.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Accessibility: How To
+---
+title: Accessibility: How To
```
Comment (dmundra) [inline practice-areas/accessibility/accessibility-how-to.md]: @grugnog looks like the title front matter doesn't like the colon in the title (I believe): https://civicactions-handbook--1123.org.readthedocs.build/en/1123/practice-areas/accessibility/accessibility-how-to/

**Review by dmundra — APPROVED** [review]
@grugnog yup I think good to go. Noticed one issue and the slashes for `$` signs (which might be needed for some parsing?) but other than that I think it is fine.

**Comment by dmundra** [comment]
@grugnog I noticed moving to front matter for the title that the materials theme was actually using the title from mkdocs.yml nav element. That is probably a good thing but something to document and keep sync'd.

**Pull request merged**